examples/features/exclude-prefixes-client/README.md

```diff
@@ -0,0 +1,282 @@
+# Test kernel to kernel connection with excluded prefixes on the client
+
+This example shows kernel to kernel example where we exclude prefixes used by 2 service endpoints on the client side. 
+
+NSC and NSE are using the `kernel` mechanism to connect to its local forwarder.
+
+## Requires
+
+Make sure that you have completed steps from [basic](../../basic) or [memory](../../memory) setup.
+
+## Run
+
+Create test namespace:
+```bash
+NAMESPACE=($(kubectl create -f https://raw.githubusercontent.com/networkservicemesh/deployments-k8s/b4e64d2c715cb540ea58400cd042412e9d67d42e/examples/use-cases/namespace.yaml)[0])
+NAMESPACE=${NAMESPACE:10}
+```
+
+Select node to deploy NSC and NSE:
+```bash
+NODE=($(kubectl get nodes -o go-template='{{range .items}}{{ if not .spec.taints  }}{{index .metadata.labels "kubernetes.io/hostname"}} {{end}}{{end}}')[0])
+```
```

Comment:
Do we need a concreate node for this example?
```suggestion
```

Reply:
Done